Clinical-Genomics/BALSAMIC
# Unit tests added for cli commands

Unit testing started. Test cases added for BALSAMIC CLI. 

How to test:
 - Change the directory to BALSAMIC - `cd BALSAMIC`
 - Pull the code from `tests` branch
 - Run this `pytest tests/` 
 
Expected outcome:
 - It should run without error.
 - 15 test cases should be passed with two warnings.
